### PR TITLE
@font-face src tech() function - add to experimental features

### PR DESCRIPTION
**Pull request by hamishwillee**
FF105 adds support for using `@font-face` `src` `tech()` function to specify the font technologies in a file, allowing a user agent to reject the src (see https://bugzilla.mozilla.org/show_bug.cgi?id=1786804)

Note, there are no associated docs. This is behind a pref because the syntax is still under discussion. We will add docs when things stabilise.

Other docs work can be tracked in #19833

### CODE REVIEW

File: files/en-us/web/css/@font-face/index.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ The `@font-face` at-rule may be used not only at the top level of a CSS, but als
 
 ## Formal syntax
 
-```
+```css-nolint
```
Comment (hamishwillee) [inline files/en-us/web/css/@font-face/index.md:109]:
```suggestion
```
```

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -637,6 +637,47 @@ See {{bug(1774589)}} for more details.
   </tbody>
 </table>
 
+### @font-face src feature checking checking
+
+The `@font-face` [`src` descriptor](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/src) now supports the `tech()` function, allowing fallback of whether a font resource is downloaded based on whether the user-agent supports a particular font feature or technology.
```
Comment (estelle) [inline files/en-us/mozilla/firefox/experimental_features/index.md:642]:
```suggestion
The `@font-face` [`src` descriptor](/en-US/docs/Web/CSS/@font-face/src) now supports the `tech()` function, allowing fallback of whether a font resource is downloaded based on whether the user-agent supports a particular font feature or technology.
```

**Review by estelle — COMMENTED** [review]
not sure if you intentionally did an absolute URL

**Review by bsmth — APPROVED** [review]
Looks good to me 👍🏻

**Pull request merged**